components/system_notice/system_notice.tsx

```diff
@@ -34,13 +34,13 @@ type Props = {
 }
 export default class SystemNotice extends React.PureComponent<Props> {
     componentDidMount() {
-        if (this.props.isSystemAdmin) {
+        if (this.props.isSystemAdmin && !this.props.analytics) {
             this.props.actions.getStandardAnalytics();
         }
     }
 
     componentDidUpdate(prevProps: Props) {
-        if (prevProps.isSystemAdmin !== this.props.isSystemAdmin && this.props.isSystemAdmin) {
+        if ((prevProps.isSystemAdmin !== this.props.isSystemAdmin && this.props.isSystemAdmin && !this.props.analytics)) {
```

Comment:
Is it possible that `this.props.analytics` is outdated and we do need the call to getStandardAnalytics to update them? Or can we assume if they're available, they're updated? I have no idea, but it seemed risky.

Reply:
for my investigation, it seems that system_notice and showStartTrialModal load at same times/circumstances, so I assume it would be safe to consider the state of the analytics would be the same for both components, however now that you mention it, I am not sure if it is worth the potential risk of system_notice getting outdated analytics information vs the benefit on not calling analytics again??